torchtune/dev/rl/workers/postprocessing.py

```diff
@@ -31,13 +31,12 @@ def __init__(self, *args, **kwargs):
         self.cfg = kwargs.pop("cfg")
         self.rollout_queue = kwargs.pop("rollout_queue")
         self.replay_buffer = kwargs.pop("replay_buffer")
-        self._device = utils.get_device(device=self.cfg.postprocessing.device_type)
+        device_type = "cuda"
```

Comment:
Why hardcode instead of reading it from cfg?

Reply:
We actually hard coded it in many different places and don't have time to test on different devices so this just explicitly sets it to "cuda"